### ISSUE TITLE
vireo error with scipy.special.binom

### ISSUE THREAD

**Issue by hongdavid94** [single-cell-genetics/vireo#46]
Hello, I am getting an error message for running vireo. 

I am not sure how to de-bug the issue, could you help me? thank you

![image](https://user-images.githubusercontent.com/27015892/169446453-52d25c70-8042-4b3d-9ab3-2e04556a0707.png)

**Comment by huangyh09** [comment]
Hi, thanks for sharing this issue. It seems an issue of incompatibility, but it's difficult to diagnose from the given error message only. 

Could you share your command line, all printed log info from Vireo and the `scipy` version (not sure if it's related)?

Yuanhua

**Comment by hongdavid94** [comment]
Hi,

the command line I used is quite simple
```
DIR_CELLSNP="~demultiplex/cellsnp_37"
DIR_VIREO="~demultiplex/vireo_37"
EXECUTE_VIREO="~env_demultiplex_37/bin/vireo"

$EXECUTE_VIREO -c $DIR_CELLSNP -N 2 -o $DIR_VIREO
```

And here is the list of modules and versions that were active during the run:
```
1) gcc/6.2.0
2) perl/5.30.0
3) miniconda3/4.10.3 (E)

Where:
E: Experimental
```

I am not sure what you mean by the printed log information, but the output file is a single line:
```
[vireo] Loading cell folder ...
```

The python, vireo, and scipy versions are:
```
Python: 3.7.12
scipy: 1.7.3
vireoSNP: 0.5.7
cellsnp-lite: 1.2.2
```

I did some investigating on my own and I realized the cellsnp-lite output had 0 variant, even though the code seemed to run without raising a flag. I am now trying to understand why a set of seemingly correct inputs (my colleague was able to use cellsnp-lite successfully with similar set of options and inputs but with a different bam file) are giving me 0 variant. I will let you know (on the cellsnp-lite github page) if the issue persists. Thank you for your quick reply
